README.md

```diff
@@ -1182,6 +1182,7 @@ Table of Contents
 
   * [AllTheFreeStock](https://allthefreestock.com) - a curated list of free stock images, audio and videos.
   * [Ant Design Landing Page](https://landing.ant.design/) - Ant Design Landing Page provides a template built by Ant Motion's motion components. It has a rich homepage template, downloads the template code package, and can be used quickly. You can also use the editor to quickly build your own dedicated page.
+  * [Atmos](https://atmos.style) - Create better UI color palettes with a specialized toolbox to find colors & create shades. Free plan limited to one project (after Beta).
```

Comment:
I dont see it stated anywhere that would be the properties of the free tier once the beta is over, if not I'd suggest coming back once its the beta is over or adding something clarifying it now